pupil-spa/src/app/virtual-keyboard/virtual-keyboard.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-virtual-keyboard',
```

Comment:
@jon-shipley I'm unsure about the scope of the story in terms of integration to the QuestionComponent. Let's discuss on Slack.